src/writer.js

```diff
@@ -380,6 +380,16 @@ Writer.prototype.bytes = function write_bytes(value) {
 };
 
 /**
+ * Writes raw bytes with no wire type of length prefixed
+ * @param {Uint8Array} value bytes to add
+ * @returns {Writer} `this`
+ */
+Writer.prototype.rawBytes = function write_raw_bytes(value) {
```

Comment:
seems that writeBytes (the one using util.Array) might be incorrect on older node versions, but it was probably never called until now